neurokit2/signal/signal_psd.py

```diff
@@ -8,14 +8,16 @@ def signal_psd(
     signal,
     sampling_rate=1000,
     method="welch",
-    show=True,
+    show=False,
+    normalization=True,
```

Comment:
I'd call it `normalize` rather than `normalization` so that's consistent with the standardize function etc.